gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -175,6 +177,29 @@ def main(args):
                 ).write(training_resources[region].path, overwrite=overwrite)
             logger.info("Done with creating training dataset.")
 
+        # Build plateau and coverage models for autosomes/pseudoautosomal regions,
+        # chromosome X, and chromosome Y
+        plateau_models = {}
+        coverage_model = {}
+        for region in GENOMIC_REGIONS:
+            logger.info("Building plateau and coverage models...")
```

Comment:
logging the region type in this logging statement might be slightly more informative than logging later on line 201, but up to your preference